lib/senkyoshi/models/file_resource.rb

```diff
@@ -19,5 +19,19 @@ def self.from(xml, pre_data, _resource_xids = nil)
     def iterate_xml(_xml, _pre_data)
       self
     end
+
+    def create_module(course)
+      course.canvas_modules ||= []
+      cc_module = Course.master_module(course)
```

Comment:
oh, I was thinking it was from course.rb. In that case we can't really do anything there. :/ bummer